### PR TITLE
[Backport] Setup Wizard: Mobile response styling

### PR DESCRIPTION
**Pull request by st0nebraker**
Manual backport based off failure [here](https://github.com/sourcegraph/sourcegraph/pull/49449#issuecomment-1472515204)

## Test plan
- Run from app to see all steps (sg start app)
- Test flow on all screen sizes (down to 320px screen width)
- Ensure no breaking UI

### CODE REVIEW

File: client/web/src/setup-wizard/components/ProgressBar.module.scss [hunk 1]
```diff
@@ -1,3 +1,16 @@
+.root {
```
Comment (coury-clark) [inline client/web/src/setup-wizard/components/ProgressBar.module.scss:1]: This diff is not the same as the original PR, have you verified this is expected on the 5.0 branch?

**Comment by sg-e2e-regression-test-bob** [comment]
## Bundle size report 📦

| Initial size | Total size | Async size | Modules |
| --- | --- | --- | --- |
| 0.06% (+1.75 kb) | 0.07% (+10.13 kb) 🔺 | 0.07% (+8.38 kb) | 0.00% (0) |

Look at the [Statoscope report](https://console.cloud.google.com/storage/browser/_details/sourcegraph_reports/statoscope-reports/backport-49449-to-5.0/compare-report.html;tab=live_object) for a full comparison between the commits 92ec805 and 530863b or [learn more](https://docs.sourcegraph.com/dev/how-to/testing#bundlesize).


<details>
<summary>Open explanation</summary>

- `Initial size` is the size of the initial bundle (the one that is loaded when you open the page)
- `Total size` is the size of the initial bundle + all the async loaded chunks
- `Async size` is the size of all the async loaded chunks
- `Modules` is the number of modules in the initial bundle
</details>

File: client/web/src/setup-wizard/components/ProgressBar.module.scss (hunk 1 above)
Comment (st0nebraker) [reply client/web/src/setup-wizard/components/ProgressBar.module.scss:1]: Ah, it's coming from [this backport](https://github.com/sourcegraph/sourcegraph/pull/49352) that hasn't merged yet. I re-tested on 5.0 branch & that CSS block is needed & works, yes!

I'm happy to wait to merge this PR until #49352 is in

Comment (coury-clark) [reply client/web/src/setup-wizard/components/ProgressBar.module.scss:1]: you're good! just want to make sure we're double checking everything is supposed to go in!

**Review by coury-clark — APPROVED** [review]

**Pull request merged**